Beginners/calculator.py

```diff
@@ -3,7 +3,7 @@
 op = input("Enter the operator [ +, -, *, /]: " )
 
```

Comment:
n1=float(input("Enter the first number:"))
```suggestion
n2=float(input("Enter the second number:"))
```